docs/topics/proc_configuring-jenkins-to-deploy-an-application-with-fmp.adoc

```diff
@@ -0,0 +1,40 @@
+= Configuring Jenkins to deploy your application with the Fabric8 Maven Plugin
```

Comment:
Do you consider the word "plugin" to be part of the name? When I write it in full form, I always write "Fabric8 Maven plugin", but maybe I'm doing it wrong :-)

Reply:
Hmm, good question. I usually write _plugin_ in this context in lower case too. I've looked at their homepage, and their docs lists it all lower case (as in _fabric8 maven plugin_), so we can't really use that as a reference point either.

Reply:
we usually say or write `fmp` or `FMP` so thats why i considered it part of the name. I don't have a strong preference if you all think it should be lower case.